manifests/piped/templates/_helpers.tpl

```diff
@@ -123,3 +123,38 @@ A set of permissions ClusterRole will contain
   - '*'
 {{- end }}
 {{- end }}
+
+{{/*
+A set of args for Launcher.
+*/}}
+{{- define "piped.launcherArgs" -}}
+- launcher
+{{- if .Values.launcher.configFromGitRepo.enabled }}
+{{- with .Values.launcher.configFromGitRepo }}
+- --config-from-git-repo=true
+- --git-repo-url={{ required "repoUrl is required" .repoUrl }}
+- --git-branch={{ required "branch is required" .branch }}
+- --git-piped-config-path={{ required "configFilePath is required" .configFilePath }}
+{{- end }}
+{{- else }}
+- --config-file=/etc/piped-config/{{ .Values.config.fileName }}
+{{- end }}
+- --metrics={{ .Values.args.metrics }}
+- --enable-default-kubernetes-cloud-provider={{ .Values.args.enableDefaultKubernetesCloudProvider }}
+- --insecure={{ .Values.args.insecure }}
+- --log-encoding={{ .Values.args.logEncoding }}
+- --add-login-user-to-passwd={{ .Values.args.addLoginUserToPasswd }}
```

Comment:
I'm now confused about how these args are passed to the piped process.

Reply:
Will the arguments specified in `spec.template.spec.containers[0].args` be passed to the appropriate process?

Reply:
Yes. Launcher will propagate all needed args to its managed Piped.
https://github.com/pipe-cd/pipe/blob/master/pkg/app/launcher/cmd/launcher/launcher.go#L430-L457

Reply:
Ah, that makes sense.

if you think about it in this way, we'd better look for another way to manage `ignroeArgs` as it's more likely to slip our mind to update this when adding a new argument to Launcher.
https://github.com/pipe-cd/pipe/blob/master/pkg/app/launcher/cmd/launcher/launcher.go#L431-L441

But it's definitely different one, great job 👍

Reply:
Right! I feel the same. Let's improve it later. 👍